integration/benchmark/server/control.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# assumes flow-go was already cloned by user
+
+# need to add this, otherwise will get the following error when systemd executes git commands
+# fatal: detected dubious ownership in repository at '/tmp/flow-go'
+git config --system --add safe.directory /opt/flow-go
+
+git fetch
+
+git log  --merges --first-parent  --format=master:%H origin/master --since '1 week' | sort -R  | tee /opt/master.recent
```

Comment:
Could we leave a comment as to the intention of this so that it is clear what information is going into the `/opt/master.recent` file?

Reply:
added explanation 1c197a1e630c984fbcb22080419d184f1d0cb4e1